python/GafferUI/NumericPlugValueWidget.py

```diff
@@ -77,7 +77,7 @@ def getToolTip( self ) :
 			if result :
 				result += "\n"
 			result += "## Actions\n"
-			result += " - Cursor up/down to increment/decrement\n"
+			result += " - Cursor or scroll wheel up/down to increment/decrement\n"
```

Comment:
Should we mention `Ctrl` + scroll wheel here?

Reply:
Ah yes, of course. I also tweaked the wording just a bit - `Cursor up/down or <kbd>Ctrl</kbd> + scroll wheel to increment/decrement`. Having "up/down" right after "cursor" feels like a clearer explanation of the keyboard part.

Reply:
That's done in a6e6cbfd415643e829e42303b7e34b29a372cf70, where I squashed everything down.